core/frontend/src/components/vehiclesetup/configuration/compass/AutoCoordinateDetector.vue

```diff
@@ -121,7 +139,21 @@ export default {
         })
         .catch((err) => console.error(err))
     },
-    setOrigin(lat: number, lon:number) {
+    async setOrigin(lat: number, lon:number) {
+      if (this.original_ekf_src === undefined) {
+        console.error('Unable to detect ekf source, aborting')
+        return
+      }
+      mavlink2rest.setParam('EK3_SRC1_POSXY', 0, autopilot_data.system_id)
+      // wait for the param to get set, wait up to 3 seconds
```

Comment:
The problem for using such comments.. is that they can be out of sync with the code.. as we are seeing now. I would recommend changing it to:
```suggestion
      // Wait for the param to get set
```

Reply:
cleaned up